src/travis2docker/templates/entrypoint_deployv.sh

```diff
@@ -101,6 +110,12 @@ def start_odoo():
     subprocess.call(cmd)
 
 
+def start_ssh():
+    if os.environ.get("START_SSH", False) and is_root():
```

Comment:
- Superfluous default value for `get`
- Since we're already adding a variable to start SSH, what do you think about adding two more ones to start PSQL and ODoo? (Whose default value would be `True`, of course)

Reply:
I guess we can add it. The ENV variables would then be something like: `NO_AUTO_ODOO`? Or what would be an appropriate name for it?

Also if `NO_AUTO_POSTGRES` is set, but Odoo is set to autostart, should we just let it naturally fail, or watch out for it?

Reply:
they would be `START_ODOO` and `START_PSQL`

ANd the code inside those methods would look something like:

```
 def start_psql():
    if not str2bool(os.environ.get("START_PSQL", True)):
        return
```

Reply:
> Also if NO_AUTO_POSTGRES is set, but Odoo is set to autostart, should we just let it naturally fail, or watch out for it?

That's a valid combination. Actually, that's the one I (and also the migration team) use, because we don't use the PSQL server inside the container but the local one. That because we need to connect to the same database from several containers.

For more info, check [this documentation](https://git.vauxoo.com/vauxoo/documentation/-/wikis/Connect-to-PostgreSQL-from-Container)